### PR TITLE
Feature/shc scanner result

### PR DESCRIPTION
**Pull request by NC-001**
- Created UI for displaying result when scanning Smart Health Card with immunization

### CODE REVIEW

File: FHICORC/Utils/DateUtils.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public static string DateToScannerResultFormat(this DateTime date)
 
         public static string ToLocaleDateFormat(this DateTime date)
         {
-            return date.ToString("d", CultureInfo.CreateSpecificCulture(LocaleService.Current.GetLanguage().ToISOCode()));
+            return date.ToString("yyyy-MM-dd", CultureInfo.CreateSpecificCulture(LocaleService.Current.GetLanguage().ToISOCode()));
```
Comment (NC-395) [inline FHICORC/Utils/DateUtils.cs:17]: This impacts DCC's decoding and presentation, doesn't it? If yes, be sure to test the impact.

**Review by NC-395 — APPROVED** [review]

File: FHICORC/Utils/DateUtils.cs (hunk 1 above)
Comment (NC-001) [reply FHICORC/Utils/DateUtils.cs:17]: Yes, but the date is still correct when scanning DCC's

File: FHICORC/ViewModels/QrScannerViewModels/QRScannerViewModel.cs [hunk 2]
```diff
@@ -241,6 +241,17 @@ private async Task OnScanSuccess(TokenValidateResultModel tokenValidateResultMod
                 anyTestResults = cwtPayload.DCCPayloadData.DCC.Tests?.Any() ?? false;
                 anyRecovery = cwtPayload.DCCPayloadData.DCC.Recovery?.Any() ?? false;
             }
+            else if (tokenValidateResultModel.DecodedModel is Core.Services.Model.SmartHealthCardModel.Shc.SmartHealthCardWrapper shc)
+            {
+                if (shc.SmartHealthCard.VerifiableCredential.CredentialSubject.Immunizations?.Any() ?? false)
```
Comment (NC-204) [inline FHICORC/ViewModels/QrScannerViewModels/QRScannerViewModel.cs:246]: What happens if we dont have any immunizations? 
Will it show an error screen, or will it do nothing? If nothing, should it show an error screen instead, so user gets feedback?

**Review by NC-204 — APPROVED** [review]

File: FHICORC/ViewModels/QrScannerViewModels/QRScannerViewModel.cs (hunk 2 above)
Comment (NC-204) [reply FHICORC/ViewModels/QrScannerViewModels/QRScannerViewModel.cs:246]: Discussed, will show invalid screen :)

**Pull request merged**